server_admin/topics/clients/oidc/proc-secret-rotation.adoc

```diff
@@ -0,0 +1,84 @@
+[id="proc-secret-rotation_{context}"]
+[[_proc-secret-rotation]]
+= Creating an OIDC Client Secret Rotation Policy
+[role="_abstract"]
+The following is an example of defining a secret rotation policy:
+
+.Procedure
+. Click *Realm Settings* in the left menu.  
+
+. Click *Client Policies* tab.
+
+. On Profiles Page, Click *Create*
++
+.Create a profile
+image:{project_images}/create-oidc-client-profile.png[Create Client Profile]
+
+. Enter any name for *Name*.
+
+. Enter a description that helps you identify the purpose of the profile for *Description*.
+
+. Click *Save*.
++
+This action creates the profile and enable to configure executors. Click *Create* to configure an executor for this profile.
```

Comment:
This action creates the profile and enables you to configure executors. 

. Click *Create* to configure an executor for this profile.